relayer/relayer.go

```diff
@@ -75,7 +77,19 @@ func (r *Relayer) route(msgs []*message.Message) {
 	}
 
 	log.Debug().Msgf("Sending messages %+v to destination %v", msgs, destChain.DomainID())
-	destChain.Write(msgs)
+
+	err := destChain.Write(msgs)
```

Comment:
Isn't `destChain.Write(msgs)` always returning `nill`?

Reply:
In this implementation it is.